### PR TITLE
fix: outline color of breakcrumb icon overflow

### PR DESCRIPTION
**Pull request by stefanoScalzo**
## Related Issue
Closes SAP/fundamental-styles#1892

## Description
Make all states have the same color for both items
### Before:
image

### After:
Screen Shot 2020-12-14 at 1 05 46 PM

#### Please check whether the PR fulfills the following requirements

1. The output matches the design specs
- [x] All values are in `rem`
- [x] Text elements follow the truncation rules
- [x] hover state of the element follow design spec
- [x] focus state of the element follow design spec
- [x] active state of the element follow design spec
- [x] selected state of the element follow design spec
- [x] selected hover state of the element follow design spec
- [x] pressed state of the element follow design spec
- [x] Responsiveness rules - the component has modifier classes for all breakpoints
- [x] Includes Compact/Cosy/Tablet design
- [x] RTL support
2. The code follows fundamental-styles code standards and style
- [x] only one top level `fd-*` class is used in the file
- [x] BEM naming convention is used
- [x] Mixins are used for repeatable code (`fd-rtl`, `fd-ellipsis`, `fd-flex`, `fd-selected`, `fd-focus`, ect.)
- [x] A11y support - keyboard support, screenreader support, proper ARIA attributes, etc.
- [x] `fd-reset()` mixin is applied to all elements
- [x] Variables are used, if some value is used more than twice.
- [x] Checked if current components can be reused, instead of having new code.
3. Testing
- [x] tested Storybook examples with "CSS Resources" `normalize` option 
- [x] tested Storybook examples with "CSS Resources" `unnormalize` option 
- [x] Verified all styles in IE11
- [x] Updated tests
4. Documentation
- [x] Storybook documentation has been created/updated
- [x] Breaking Changes [wiki](https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes) has been updated in case of breaking changes.

### CODE REVIEW

**Review by JKMarkowski — APPROVED** [review]

File: src/breadcrumb.scss [hunk 1]
```diff
@@ -54,6 +54,30 @@ $fd-breadcrumb-border-top-radius: 0.125rem !default;
...
+      &:hover {
+        .#{$block}__dropdown-icon {
+          color: var(--sapLink_Active_Color, #0a6ed1);
+        }
+      }
+    }
+
+    &:hover {
+      .#{$block}__dropdown-icon {
+        color: var(--sapLink_Hover_Color, #0854a0);
```
Comment (InnaAtanasova) [inline src/breadcrumb.scss:78]: Why are you adding the colors this way? 
Should be just color: `var(--sapLink_Hover_Color);` or `color: var(--sapLink_Active_Color)

Comment (InnaAtanasova) [reply src/breadcrumb.scss:78]: and since `var(--sapLink_Active_Color)` is repeated 3 times maybe put it in a variable.

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]

File: src/breadcrumb.scss (hunk 1 above)
Comment (stefanoScalzo) [reply src/breadcrumb.scss:78]: I noticed we have fallbacks in some components but ill change it :)

**Review by InnaAtanasova — APPROVED** [review]

File: src/breadcrumb.scss (hunk 1 above)
Comment (InnaAtanasova) [reply src/breadcrumb.scss]: That's not the proper way to add a fallback in general. You will need:
```
color:  #0854a0;
color: var(--sapLink_Hover_Color, #0854a0);
```

Comment (InnaAtanasova) [reply src/breadcrumb.scss]: Also, I searched the project and couldn't find such case :)
There's one usage in `custom.scss` file which is for the documentation. I think you are talking about fundamental-ngx, but this is another case.

**Pull request merged**